thorium/testsuite/test_fields.py

```diff
@@ -426,6 +426,25 @@ def test_get_validator(self):
         self.assertRaises(NotImplementedError, self.field._get_validator)
 
 
+class TestDefaultProperty(TestCase):
+
+    def setUp(self):
+        self.field = SimpleResourceField()
+
+    def test_default_with_function(self):
+        def default_func():
+            return 5
```

Comment:
Something like a timestamp or guid generation would be a better test here, that way you can make sure that it's not the same value every time a new resource is created.